vcap/setup.py

```diff
@@ -5,11 +5,24 @@
 
 setup(
     name='vcap',
-    version='0.1.3',
     description="A library for creating OpenVisionCapsules in Python",
+    author="Dilili Labs",
     packages=find_namespace_packages(include=["vcap*"]),
 
-    author="Dilili Labs",
+    # Pull the package version from Git tags
+    use_scm_version={
```

Comment:
Does this require the user to have setuptools_scm pre-installed, or is this something that is supported by pip/setuptools somehow?

Reply:
I didn't have to pip install setuptools_scm myself, so I think it's managed by setuptools. I just tried it in a fresh virtualenv and it worked fine. Not sure exactly how this works internally though.

Reply:
Awesome!